deployments/ansible/roles/collector/README.md

```diff
@@ -105,8 +126,50 @@ how to use the role in a playbook with minimal required configuration:
 - `splunk_fluentd_config`: Path to the fluentd config file on the remote host.
   (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
 
+## Windows Specific Role Variables
+
+- `collector_path_win`: Default path of Splunk-otel-collector in windows 
+  (**default:** `C:\\Program Files\Splunk\OpenTelemetry Collector\`)
+
+- `splunk_bundle_dir_win` & : The path to the [Smart Agent bundle directory](
+  https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/README.md).
+  The default path is provided by the collector package. If the specified path
+  is changed from the default value, the path should be an existing directory
+  on the node. The `SPLUNK_BUNDLE_DIR` environment variable will be set to
+  this value for the collector service.  (**default:** `${collector_path_win}\agent-bundle`)
+
+- `splunk_collectd_dir_win`: The path to the collectd config directory for the
+  Smart Agent bundle. The default path is provided by the collector package.
+  If the specified path is changed from the default value, the path should be
+  an existing directory on the node. The `SPLUNK_COLLECTD_DIR` environment
+  variable will be set to this value for the collector service.
+  (**default:** `${splunk_bundle_dir_win}\run\collectd`)
+
 - `splunk_fluentd_config_source`: Source path to a fluentd config file on your 
   control host that will be uploaded and set in place of `splunk_fluentd_config` on
   remote hosts. Can be used to submit a custom fluentd config,
   e.g. `./custom_fluentd_config.conf`. (**default:** `""` meaning 
   that nothing will be copied and existing `splunk_fluentd_config` will be used)
+
+- `collector_config_source_win`: Source path to the collector config YAML file. This file will 
+  be copied to the $collector_config_dest path on the node. See the source attribute of the file 
```

Comment:
The `See the source attribute of the file resource for supported value types.` statement is not applicable to ansible and should be removed.